drone-webdriver/src/test/java/org/jboss/arquillian/drone/webdriver/binary/downloading/source/GitHubSourceLatestReleaseFromTagsTestCase.java

```diff
@@ -30,6 +32,8 @@
 import static org.mockito.Mockito.times;
 import static org.mockito.Mockito.verify;
 
+@Ignore("For the evaluation the PhantomJSGitHubBitbucketSource is used. But it doesn't use this logic any more."
```

Comment:
how about this: https://github.com/arquillian/arquillian-extension-drone/pull/88/commits/fc78f3c01ef916b13e3a1ade15b66c491e2fbe2c#diff-00a1f71655dc18c79ab7c7f8c0735aa3R180